tests/StringParseTest.cpp

```diff
@@ -26,30 +26,6 @@ TEST_F(TApp, ExistingExeCheck) {
     EXPECT_EQ(str3, "\"quoted string\"");
 }
 
-TEST_F(TApp, ExistingExeCheckWithSpace) {
```

Comment:
Why is this being removed?

Reply:
This was mainly testing the file name separation, which is then redundant with the new Validators test since the function that handles that was moved out of the parse function.

Reply:
I think having an integration-style test in addition to the unit-style test is not a bad idea, at least as long as it already exists. I might add it back in.